### PR TITLE
LUN-102: Mapping for articles of alternative words

### PR DESCRIPTION
**Pull request by SteffiStuffel**
This pull request belongs to an issue on our [bugtracker](https://issues.tuerantuer.org/secure/RapidBoard.jspa?rapidView=27&view=detail).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword _LUN_.

### CODE REVIEW

**Review by Taggotty — APPROVED** [review]

File: src/hooks/useLoadDocuments.ts [hunk 1]
```diff
@@ -1,31 +1,34 @@
...
+  if (documents.data) {
+    formattedDocuments = documents.data?.map(document => ({
+      ...document,
+      article: ARTICLES[document.article],
+      alternatives: document.alternatives.map(it => ({
+        article: ARTICLES[it.article],
+        word: it.alt_word
+      }))
+    }))
+  }
```
Comment (klinzo) [inline src/hooks/useLoadDocuments.ts:30]:
```suggestion
  const formattedDocuments: DocumentType[] = documents.data?.map(document => ({
      ...document,
      article: ARTICLES[document.article],
      alternatives: document.alternatives.map(it => ({
        article: ARTICLES[it.article],
        word: it.alt_word
      }))
    })) ?? []
```
should work as well.

File: src/routes/write-exercise/components/__tests__/AnswerSection.spec.tsx [hunk 2]
```diff
@@ -1,11 +1,44 @@
 import React from 'react'
 import AnswerSection, { AnswerSectionPropsType } from '../AnswerSection'
-import { shallow } from 'enzyme'
-import toJson from 'enzyme-to-json'
+import { fireEvent, render } from '@testing-library/react-native'
+import labels from '../../../../constants/labels.json'
+
+jest.mock('../../../../components/AudioPlayer', () => {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const Text = require('react-native').Text
```
Comment (klinzo) [inline src/routes/write-exercise/components/__tests__/AnswerSection.spec.tsx:8]: Can't we just import it normally?

**Review by klinzo — APPROVED** [review]

**Comment by klinzo** [comment]
Tested on android.

**Review by klinzo — APPROVED** [review]
:tada:

**Pull request merged**